src/assets/scripts/client/ui/TrafficRateController.js

```diff
@@ -98,6 +102,7 @@ export default class TrafficRateController {
 
         this._buildDialogBody();
         this.$element.append(this.$dialog);
+        this.$dialog.find('#reset-button').click(this._onFormReset)
```

Comment:
ugh, bad Nate!  more sleeping on PRs. 

when we need to access DOM elements via js, usually, we use a classname prefixed with `js-`. That we we can easily tell its a hook for js events and shouldn't be used for styling. Additionally, this selector should probably live with the rest of them here: https://github.com/openscope/openscope/blob/master/src/assets/scripts/client/constants/selectors.js

Reply:
@n8rzz  I am using the element id here as the selector, should I be using a js- classname instead?

Reply:
yes please

Reply:
and then reference that using the `SELECTORS` constant from the file I linked above